addons/t9n/models/message.py

```diff
@@ -11,14 +11,67 @@ class Message(models.Model):
     _description = "Localizable message"
 
     body = fields.Text(
+        "Entry to be Translated",
         help="The actual, textual content to be translated.",
     )
+    context = fields.Char(
+        help="Context of the message.",
+    )
+    translator_comments = fields.Text(
+        help="Comments wrote by the translator/developer in the resource file.",
+    )
+    extracted_comments = fields.Text(
+        help="Comments extracted during the process of extracting translatable strings.",
+    )
```

Comment:
I'm not sure if this is the best name for it, but it makes it clearer that they are different from the translator comments

```suggestion
    extracted_comments = fields.Text("Resource Comments")
```